test/fw/bin/pbs_benchpress

```diff
@@ -120,6 +116,8 @@ def usage():
     msg += ['--min-pyver=<version>: minimum Python version\n']
     msg += ['--max-pyver=<version>: maximum Python version\n']
     msg += ['--param-file=<file>: get params from file. Overrides -p\n']
+    msg += ['--post-analysis-data=<dir>: path to post analysis data' +
+            ' directory\n']
```

Comment:
Add tc_failure_threshold,  cumulative_tc_failure_threshold and  max_postdata_threshold info to pbs_benchpress help